app/views/contacts/edit.html.haml

```diff
@@ -0,0 +1,20 @@
+.row.justify-content-center
+  %h2.title= t('.contacts_edit')
+.row
+  .col-lg-5.col-md-5
+    = simple_form_for @contact, as: @contact do |f|
+      = f.input :email, label: 'Email', autofocus: true, placeholder: 'example@mail.com'
+      = f.input :phone, label: t('.phone')
+      = f.simple_fields_for :address do |a|
+        = a.input :latitude, as: :hidden
+        = a.input :longitude, as: :hidden
+      = label_tag 'searchTextField', t('.change_address')
+      %input#searchTextField.form-control.is-valid.string{placeholder: t('.location'), type: "text"}/
+      .actions
+        = button_tag 'change address', type: 'button', class: 'btn btn-outline-info btn-size'
+        = f.button :submit, t("create"), class: 'btn btn-outline-info btn-size'
+  .col-lg-7.col-md-7.map-align
+    = content_tag :div, nil, id: 'map'
+    :javascript
+      $('button').click(()=>{ ChangeAddress();});
```

Comment:
bad practice to write JS code in the view
move it to js file and add unique `id` to the button